src/d3d9/d3d9_device.cpp

```diff
@@ -2945,13 +2945,13 @@ namespace dxvk {
          slice = slice.subSlice(offset, slice.length() - offset);
 
     EmitCs([this,
-      cDecl          = ref(decl),
-      cVertexCount   = VertexCount,
-      cStartIndex    = SrcStartIndex,
-      cInstanceCount = GetInstanceCount(),
-      cBufferSlice   = slice
+      cVertexElements = decl->GetElements(),
```

Comment:
This is kind of not ideal since `D3D9VertexElements` is a vector, but unless games spam this thousands of times per frame it *should* be fine?

Reply:
I know but the only game I know off the top of my head that uses ProcessVertices and that's Sims 2. So performance wasn't the greatest priority.

Reply:
FWIW, it's a bit more common in d3d8. I've seen it show up in quite a few traces, not quite at the levels of thousands per frame, but there's definitely some occasional heavy use of it out there as well.